utils/src/main/scala/za/co/absa/enceladus/utils/time/TimeZoneNormalizer.scala

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2018-2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.utils.time
+
+import java.util.TimeZone
+
+import com.typesafe.config.{Config, ConfigFactory}
+import org.apache.log4j.{LogManager, Logger}
+import org.apache.spark.sql.SparkSession
+
+/**
+  * Sets the system time zone per application configuration, recommended value being UTC
+   */
+object TimeZoneNormalizer {
+  private val log: Logger = LogManager.getLogger(this.getClass)
+  private val timeZone: String = getConf("timezone", "UTC")
+  private var justBuilt: Boolean = false
+
+  setJVMTimeZone()
+  justBuilt = true
+
+  private def getConf(path: String, default: String): String = {
+    val config: Config = ConfigFactory.load()
+    if (config.hasPath(path)) {
+      config.getString(path)
+    } else {
+      log.warn(s"No time zone (TimeZone) setting found. Setting to default, which is $default.")
+      default
+    }
+  }
+
+  private def setJVMTimeZone(): Unit = {
+    if (justBuilt) {
+      justBuilt = false
```

Comment:
I don't understand what this method is doing. It is sets the timezone once on initialization, and prevents doing it again the second time it is called, but resets the `justBuilt` flag to false (which never gets sets to true again) and I can keep setting the timezone as many times as I want after that. 
Ultimately calling it more than once changes nothing, since every time I call this method it always sets the timezone to `UTC`, so I don't know why we are trying to prevent repeated calls. Is `TimeZone::setDefault` a heavy operation?